src/storage.cc

```diff
@@ -75,9 +75,16 @@ void Storage::EnterNotify() {
   buy_policy.Start();
 
   if (out_commods.size() == 1) {
-    sell_policy.Init(this, &stocks, std::string("stocks"))
+    if (sell_quantity != 0) {
+      sell_policy.Init(this, &stocks, std::string("stocks"), 1e+299, false, sell_quantity)
```

Comment:
Do we have 1e+299 defined as a canonical large number in cyclus anywhere?  

I'm also wondering if we should use variables to communicate what these values are, i.e. avoid magic numbers.


```suggestion
      max_throughput = 1e+299;
      ignore_composition = false;
      sell_policy.Init(this, &stocks, std::string("stocks"), max_throughput, ignore_composition, sell_quantity)
```

Reply:
I created #1553 in Cyclus to worry about this.